ConnorAndersonLarson/PlanetParty
# Feature/all planets view

# PR Rancid Tomatillos
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ X ] Feature

### Are there any known bugs?

- [ X ] No

### Summary of the changes
Add a Planet and Planetarium component with tsx and css files. Create basic planet card structure from test data to display planet image, planet name, and planet fact on card. Also create basic css styling for Planetarium and Planet component.


### What ticket(s) do the changes resolve?

#11 

Please Link Issues to this PR as necessary.

### Notes for the Reviewer

This is only using test data for now. Feel free to adjust interface and className names.


### How to Test Changes?

Pull down this branch, run `npm i` then `npm start`. All 4 test data planets should appear on the DOM.

### Next Iterations

Create an image folder holding all planet images, link image maybe by id to planet card. Create a function to display planet fact, i.e. `Distance from the Sun` or `Average Temperature` -- needs to be dynamic.
